### DISCUSSION TITLE
foundry section 2  alwaays getting no src file of FundME.sol

### DISCUSSION THREAD

**Discussion by guruprakash01** [Cyfrin/foundry-full-course-cu#1384] [Q&A]
help me im getting no src file found error alwaysCompiler run failed:

Error (6275): Source "src/fundMe.sol" not found: File not found. Searched the following locations: "/root/foundry-f-me-f23".
 --> test/FundMeTest.t.sol:5:1:
  |

**Comment by tobezhanabi** [comment]
chck your source file, you are missing something

**Comment by heheboii11** [comment]
please send the screenshot of your vs code
